### DISCUSSION TITLE
希望加入一键发送剪切板文本的功能

### DISCUSSION THREAD

**Discussion by itribs** [localsend/localsend#605] [L-简体中文]
电脑跟手机都可以加在状态栏菜单上，这样可以实现基本的剪切板同步了

**Comment by Mapaler** [comment]
我赞同加入这个功能，比如二维码制作器就有直接使用剪切板内容的功能。
![Screenshot_2023-09-19-21-14-56-584_com teacapps barcodescanner](https://github.com/localsend/localsend/assets/6565860/1d5b84d7-4da8-49a0-94bb-8c91dd67afae)
可以多加一个按钮，需要实现纯文本、剪切板中的图片（来自截图）、文件（来自复制的文件）三种情况。
![localsend_app_2023-09-19_22-36-02](https://github.com/localsend/localsend/assets/6565860/afeb8984-99f4-46c9-8ee8-86f31fb87c90)

最好还增加QQ等聊天工具的逻辑，先选定设备，然后可以直接粘贴任意文件、图片、文本内容的形式。
![TIM_2023-09-19_22-24-49](https://github.com/localsend/localsend/assets/6565860/61f5bf05-c4b4-439b-bc88-3841b79ab8eb)
